try{
    console.log("try에 있는 문장");
}
catch{

}
finally{

}